phpunit/class-admin-test.php

```diff
@@ -75,6 +75,20 @@ function test_gutenberg_can_edit_post() {
 
 		wp_set_current_user( self::$editor_user_id );
 		$this->assertTrue( gutenberg_can_edit_post( $generic_post_id ) );
+
+		$blog_page_without_content = self::factory()->post->create( array(
+			'post_title'   => 'Blog',
+			'post_content' => '',
+		) );
+		update_option( 'page_for_posts', $blog_page_without_content );
```

Comment:
Are these tests isolated from one another? Is this something which should be reverted in a `tearDown` ?

Reply:
@aduth All MySQL queries in WordPress unit tests use the `COMMIT` and `ROLLBACK` approach to ensure that things like `update_option` are automatically rolled back. See https://github.com/WordPress/wordpress-develop/blob/master/tests/phpunit/includes/testcase.php#L155